apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -0,0 +1,402 @@
+      src: "../../../assets/img/components/tag/modifier-emphasis-strong.svg",
+      width: "49px",
+      alt: "A variation of tag which uses strong colors."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Subtle</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-emphasis-subtle.svg",
+      width: "49px",
+      alt: "A variation of tag which uses subtle colors."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Icon
+
+Icons can be used to visually support the Tag’s label. Only available at the large size.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>With icon</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-icon.svg",
+      width: "69px",
+      alt: "A tag with an icon."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Without icon</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-no-icon.svg",
+      width: "49px",
+      alt: "A tag without an icon."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+--- 
+
+## Sizes
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>Small</h3>
+    <p>16px height. Icon isn’t available at the small size option.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/size-small.svg",
+      width: "37px",
+      alt: "Small size tag with 16px height."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Large</h3>
+    <p>24px height.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/size-large.svg",
+      width: "126px",
+      alt: "Large size tag with 24px height."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## States
+
+Tag is available in 2 states. The disabled state has an opacity of 50%.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>Default</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/state-default.svg",
+      width: "126px",
+      alt: "Default state of a tag."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Disabled</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/state-disabled.svg",
+      width: "126px",
+      alt: "Disabled state of a tag."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+--- 
+
+## Content
+
+### Label guidance
+
+Tag doesn’t have a maximum width or overflow. The container width is dynamic to the content’s width. 
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use label with concise and scannable text.",
+            "Use no more than 3 words in both Large and Small tag labels.",
+            "Use sentence case in the label."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don't overcrowd the label with unnecessary information or excessive details.",
+            "Don't use more than 3 words in the label."
```

Comment:
```suggestion
            "Include unnecessary information or excessive detail.",
            "Use more than 3 words."
```

Reply:
Question for design, I guess. Should each sentence start with "Don't" or is that already implied from the context?

Reply:
We typically reinforce the "don't" at the start of each sentence. You can also use "Do not".

Reply:
I will keep the original  copy for consistency with other components